docs/API.md

```diff
@@ -367,6 +379,52 @@ Example response:
 An array of *Profile names* in lower case. `"default"` is always included.
 
 
+## API method: `get_language_tags`
+
+Returns valid [language tags][language tag] generated from the setting in the
+configuration file.
```

Comment:
The most important thing here is that the method returns the complete set of valid tags. How it is calculated is secondary.

Reply:
It is relevant information.